VMWare/vmware-vcenter/ingest/parser.yml

```
@@ -0,0 +1,78 @@
+          HTTP_LOGS_1: '%{TIMESTAMP_ISO8601:timestamp} %{DATA:log_level} %{DATA:process}\[%{DATA:pid}\] \[%{DATA:originator_info}\] %{TIMESTAMP_ISO8601:datetime} %{DATA:http_method} %{URIPATH:url_path} HTTP/%{NUMBER:http_version} %{NUMBER:response_status} %{DATA:upstream_status} - %{NUMBER} %{NUMBER} %{NUMBER} %{NUMBER} %{NUMBER} %{IP:source_ip}:%{NUMBER:client_port} %{IP:destination_ip}:%{NUMBER:server_port} %{IP:local_ip1}:%{NUMBER:local_port1} %{IP:local_ip2}:%{NUMBER:local_port2}'
+          HTTP_LOGS_2: '\[%{HTTPDATE:timestamp}\] %{DATA:process} %{IP:client_ip}:%{NUMBER:client_port} to local %{NUMBER:destination_port} - HTTP/%{NUMBER:http_version} %{DATA:http_method} %{URIPATH:url_path} %{DATA} %{NUMBER:response_status} - %{NUMBER:response_bytes} bytes %{DATA} process %{NUMBER:process_time}ms / commit %{NUMBER:commit_time}ms / conn \[%{DATA:conn_status}\]'
+          OTHERS_EVENTS: '%{TIMESTAMP_ISO8601:timestamp} %{DATA:log_level} %{DATA:process}\[%{INT:pid}\] \[%{DATA}\] %{DATA} -- %{GREEDYDATA:reason}'
+          OTHERS_EVENTS_TYPE_2: '%{TIMESTAMP_ISO8601:timestamp} \[%{DATA:thread}\] %{DATA:log_level} %{DATA:operationID} %{DATA:process} - %{GREEDYDATA:reason}'
+          OTHERS_EVENTS_TYPE_3: '%{TIMESTAMP_ISO8601:timestamp}: %{DATA}: \[%{GREEDYDATA:reason}'
+          OTHERS_EVENTS_TYPE_4: '%{GREEDYDATA:reason}'
+          OTHERS_EVENTS_TYPE_5: '\[%{TIMESTAMP_ISO8601:timestamp}\] \[%{DATA:log_level} \] %{GREEDYDATA:reason}'
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parsed_event.message.timestamp}}"
+          user.name: "{{parsed_event.message.username}}"
+          source.ip: "{{parsed_event.message.source_ip}}"
+          destination.ip: "{{parsed_event.message.destination_ip}}"
+          source.user.name: "{{parsed_event.message.source_user_name}}"
+          user_agent.original: "{{parsed_event.message.user_agent}}"
+          host.name: "{{parsed_event.message.hostname}}"
+          event.reason: "{{parsed_event.message.reason}}"
+          log.level: "{{parsed_event.message.log_level}}"
+          http.request.method: "{{parsed_event.message.http_method}}"
+          url.path: "{{parsed_event.message.url_path}}"
+          http.response.status_code: "{{parsed_event.message.response_status}}"
+          http.response.bytes: "{{parsed_event.message.response_bytes}}"
+          vmware_vcenter.event_id: "{{parsed_event.message.id}}"
+          event.code: "{{parsed_event.message.event_code}}"
+          process.pid: "{{parsed_event.message.pid}}"
+          process.end: "{{parsed_event.message.process_time}}"
+          process.name: "{{parsed_event.message.process}}"
+          client.port: "{{parsed_event.message.client_port}}"
```

Useless according to changes in patterns
```suggestion
          source.port: "{{parsed_event.message.source_port}}"
```